src/quantifiers/repeat.ts

```diff
@@ -5,11 +5,11 @@ export function compileRepeat(
   config: RepeatConfig,
   compiledChildren: string
 ): string {
-  if ('count' in config && typeof config.count === 'number') {
+  if ('count' in config) {
     return `${wrapGroup(compiledChildren)}{${config.count}}`;
   }
 
-  if ('min' in config && typeof config.min === 'number') {
```

Comment:
Why did you remove it?

Reply:
<img width="753" alt="image" src="https://github.com/callstack/ts-regex/assets/22678765/71836f6c-e69d-48c5-9389-308d72722955">

Reply:
In which case we'll hit last return? Based on the config type it seems that it will always hit first or second return.